### ISSUE TITLE
Allow hosts to add a manual payment method for orders

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#1268]
Now that we support pending orders, we should allow hosts to add a manual payment method.

Use case:
AirBnB wants to give $10k to Webpack, they won't process such payment on a credit card. They want to do a wire. They should be able to select "wire" as a payment method and receive the bank details of the host. Then the host should be able to manually mark that order as paid when the money has been received on their bank account.

**Comment by Betree** [comment]
This has been implemented already

**Comment by xdamman** [comment]
But we still need to add this in the edit host collective page. Right now
it’s a manual edit oh hostCollective.settings in the database.

**Comment by Betree** [comment]
@xdamman Got it, I'm re-opening and updating the labels

**Comment by piamancini** [comment]
We will be needing this for the new expense flow, it'd be great to tackle it during the hackathon

**Comment by xdamman** [comment]
This should be linked to the new business model. People should pay $10/month to be able to accept donations directly on their bank account with the wire transfer/pending order feature.

As a quick implementation to unblock some users and also to already start getting feedback on that feature and pricing, this is what I'd suggest:

![](https://d.pr/free/i/K2852Q+)

We could link to the documentation page for the latest pricing regarding hosts while we update the current [pricing page](https://opencollective.com/pricing).

And I'd create a new $10/month tier on https://opencollective.com/opencollective for "Open Collective Pro" (by lack of better name, we can always change it later) and redirect people to the contribution flow for that tier.

I'd leave for another issue the part for actually enforcing becoming a member of that tier to use that feature after $1,000 received.

**Comment by piamancini** [comment]
couple of thoughts:
- It should be in the same place where you connect Stripe. It's weird that you are mixing it with a payout method instead of the other pay-in method.
- Is the + sign taking you to a place where you can add the copy from the email to be sent? If yes, it needs to be super clear.
- I think contextual FAQ is needed. Right now, I would expect almost like integrating a bank account, not sending an email and creating a pending order I need to manually check.

Please call it Open Collective Monthly Tier (don't use PRO)

> I'd leave for another issue the part for actually enforcing becoming a member of that tier to use that feature after $1,000 received.

Please open this issue and start describing it if you are going to own the enforcement of this policy. So we don't end up doing a bunch of support for an issue no one is working on / owning.

**Comment by xdamman** [comment]
Bear in mind that is meant as a quick fix compared to what we have now:
![](https://d.pr/free/i/7lreN6+)
- Right now Stripe is in “Connected Accounts” alongside Twitter which I think is fine for now but should be ideally moved or duplicated to the Payment Methods settings pane. Reason being that well, those are all “Payment Methods”, i.e. ways for receiving or sending money. So it’s more Stripe that should be moved here. The fact that it is right now in “Connected Accounts” alongside Twitter is a very engineer centric view (because they are both accounts connected via oAuth to your open collective account). But that’s beyond the scope of this.
![](https://d.pr/free/i/C1LdPl+)
- The button will take you to a form where you can add your bank account details and private instructions that are sent by email. Given that you are not modifying anything yet, it’s safe for people to click to discover what’s behind.
- Yes contextual FAQ is needed but it’s not designed yet for those pages. That’s why for now I’ve added help text below each button.

**Comment by piamancini** [comment]
> So it’s more Stripe that should be moved here
Please also open an issue about this with you as the owner. Otherwise it will never happen and the fix like this is confusing. You can then assign it to someone else. But you should own this feature. 

> That’s why for now I’ve added help text below each button. 
Ok so maybe explain this properly in the form. 
We are not enabling to receive donations automatically. So maybe the button should read, to send your account details to your contributors and receive donations.... ?

**Comment by xdamman** [comment]
![](https://d.pr/free/i/wcIJ5g+)

![](https://d.pr/free/i/9T9sx6+)

**Comment by alanna** [comment]
We definitely need this feature. Maybe it makes more sense to move Stripe to 'payment methods' too. In my ideal world, the host would be able to freely define the options here. For example, bitcoin, paypal (direct, not via contribution flow), cash, etc. For each, you'd just need a free text box where the host can input details, so I think it could be generalisable (let them set the payment method name too). But I'm happy to keep this to adding the wire transfer option only for now if it's best to keep it simple.

**Comment by piamancini** [comment]
Maybe enable Contributors? 
We are not using the word donors anywhere else.

**Comment by piamancini** [comment]
Also, why "kindly ask you"? Are we doing this optional?

**Comment by xdamman** [comment]
@alanna yes, would totally love to add paypal and possibly other payment manual methods (e.g. Venmo). Since the host has to manually confirm reception of the money, it is very versatile. Then based on usage we can prioritize automating each of them.
